sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_sentiment.py

```diff
@@ -121,15 +137,6 @@ def test_bad_credentials(self, client):
                 ["This is written in English."]
             )
 
-    @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer()
-    def test_bad_model_version(self, client):
```

Comment:
are we deleting because the test is duplicated?

Reply:
yep, we had test_bad_model_version_error and test_bad_model_version. Deleting test_bad_model_version since test_bad_model_version_error goes more into the returned error and is more detailed